Helper/FeatureSwitch/Decider.php

```diff
@@ -156,7 +161,19 @@ public function isSwitchEnabled($switchName) {
      * Switch Helpers below
      ***************************************************/
 
-    public function isSampleSwitchEnabled() {
+    /**
+     * Checks whether the sample feature switch is enabled
+     */
+    public function isSampleSwitchEnabled()
+    {
         return $this->isSwitchEnabled(Definitions::M2_SAMPLE_SWITCH_NAME);
     }
-}
\ No newline at end of file
+
+    /**
+     * Checks whether the feature switch for saving/sending tracking data is enabled
+     */
```

Comment:
Please add `@return bool` to PHPDoc Comment

Reply:
this too